Netty no flush

akurilov opened the pull request.

Commit:
changes to test 1st...

Signed-off-by: Andrey Kurilov <andrey.kurilov@emc.com>

Commit:
changes to test 1st...

Signed-off-by: Andrey Kurilov <andrey.kurilov@emc.com>

Commit:
try to rollback even more

Commit:
try to rollback even more

Commit:
try to rollback even more

dadlex commented on build.gradle:
```diff
@@ -57,7 +57,7 @@ ext {
 		log4j                    : "2.8.2",
-		mongooseBase             : "4.2.16",
+		mongooseBase             : "4.2.17",
 		mongooseStorageDriverPreempt: "4.2.21",
```
Why do we rollback the preempt version?

dadlex replied:
Should this be 4.2.23?

akurilov replied:
rollback is for testing

Commit:
try to rollback even more

Commit:
try to rollback even more

Commit:
try to rollback even more

Commit:
try to rollback even more

Commit:
try to rollback even more

Commit:
try to rollback even more

Commit:
quick test

Commit:
quick test 1

Commit:
quick test 2

Commit:
quick test 3

Commit:
quick test 4

Commit:
Rollback to 4.2.16 while 4.2.17 has a bug
Switch back to master branch as netty-no-flush was merged
Set the pravega commit to the one with the netty changes merged
Set version to the closest release before netty (it still doesn't make a lot of sense)

Commit:
Switch test configs to suit 4.2.16 base

Commit:
Switch test configs to suit 4.2.16 base

Commit:
Revert the version of preempt driver to to the bug in 4.2.23 (BASE-1396)
Switch timeout control-timeoutMillis to 2000ms as waiting 10sec is obnoxious

Commit:
try creating less streams in the tests for byteStreams to decrease the chance of gitlab ci timeout failure

Commit:
Avoid working with system pravega streams

Commit:
Maven2 moved to https

Commit:
Try using explicit step id for readAllByteStreams test

We call listStreams method and inside this method at some point we call parseLong java method with parameters step id (like 20200116.165417.827) and radix =36. This causes NumberFormatException

Commit:
Revert last change

Commit:
Try avoiding using an env var in test

Commit:
Add debug messages to run in gitlab ci

Commit:
Add debug messages to run in gitlab ci

Commit:
Add debug messages to run in gitlab ci

Commit:
Omit 1 test temporarily

Commit:
add tag in gitlab-ci.yml to use specific runner

Commit:
delete excessive keyword in gitlab-ci.yml

Commit:
delete tags in gitlab-ci.yml

Commit:
Update mongoose-base version with fixed log appending, rest distributed calls and count-limit fix

Commit:
Update test and set back the parameter added in 4.2.17

Commit:
Add parameters needed for 4.2.21 preempt to not fail with memory errors

Commit:
Add ps output to check that pravega standalone is actually running on a shared runner while performing one of the tests

Commit:
Correct robot test

Commit:
Delete unknown flag -u for busybox 1.31.1

Commit:
Add tags back to debug on a specific runner

Commit:
state docker version explicitly
set privileged mode on gitlab runner to true

Commit:
Try running pravega in the docker

Commit:
Correct docker run command

Commit:
Correct docker run command

Commit:
Correct docker run command

Commit:
Correct docker run command

Commit:
Try running mongoose in docker

Commit:
Try running mongoose in docker

Commit:
Try running mongoose in docker

Commit:
Try running mongoose in docker

Commit:
Add tests back
Run mongoose through robot framework, run pravega in docker

Commit:
Add tests back
Run mongoose through robot framework, run pravega in docker

Commit:
test running pravega manually for integration tests

Commit:
Try running all 5 tests

Commit:
Try using faster preempt

Commit:
Try running integration tests with pravega in docker

Commit:
Try running integration tests with pravega in docker

Commit:
Turn off auth in the integration tests

Commit:
Leave integration tests run with pravega run manually

Commit:
exclude read_all_byte_streams test until preempt bug is fixed

Commit:
Temporarily suspend byteStream support due to new version of the preempt

Commit:
Set default value for 4.2.17 base parameter to false

veronikaKochugova commented on src/test/robot/api/storage/pravega.robot:
```diff
@@ -36,7 +37,7 @@ Create Event Stream Test
 
 Create Byte Streams Test
     [Tags]  create_byte_streams
-    ${node_addr} =  Get Environment Variable  SERVICE_HOST  127.0.0.1
```
Why did you use the ip instead of SERVICE_HOST variable?

dadlex replied:
I brought back the SERVICE_HOST variable, but set it to localhost as it's the ip we use to connect to pravega that now runs in docker

Commit:
Leave SERVICE_HOST configurable
